src/GitHub.App/Services/PullRequestEditorService.cs

```diff
@@ -1,25 +1,195 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.ComponentModel.Composition;
+using System.IO;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Reactive.Threading.Tasks;
+using System.Threading.Tasks;
+using GitHub.Commands;
+using GitHub.Extensions;
+using GitHub.Models;
+using GitHub.VisualStudio;
 using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Editor;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Projection;
 using Microsoft.VisualStudio.TextManager.Interop;
-using GitHub.Models;
+using Task = System.Threading.Tasks.Task;
 
 namespace GitHub.Services
 {
+    /// <summary>
+    /// Services for opening views of pull request files in Visual Studio.
+    /// </summary>
     [Export(typeof(IPullRequestEditorService))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
     public class PullRequestEditorService : IPullRequestEditorService
     {
-        readonly IGitHubServiceProvider serviceProvider;
-
         // If the target line doesn't have a unique match, search this number of lines above looking for a match.
         public const int MatchLinesAboveTarget = 4;
 
+        readonly IGitHubServiceProvider serviceProvider;
```

Comment:
It looks like this is no longer being used.

Reply:
It is still being used!